### ISSUE TITLE
Column names: "_" produces an extraneous comma

### ISSUE THREAD

**Issue by jeromenerf** [go-pg/pg#464]
Following https://github.com/go-pg/pg/pull/428 and #422 and updating to the latest git revision (`5fde4b65f7338b401e44e90495a9cba69478fa45`): 

Given the following code that is meant to get some `User`'s with the "WORKER" `role` for a given `Contract` :

```go
		// Get some contract's workers
		users := []User{}
		err = app.DB.
			Model(&[]ContractUser{}).
			Column("_", "User").
			Where(`"contract_user".contract_id = ? AND "contract_user".role = 'WORKER'`, contract.ID).
			OrderExpr("random()").
			Limit(1 + rand.Intn(5)).
			Select(&users)

```

I use the `ContractUser` relation and omit its columns to return `User`s by using  `Column("_", "User")` but the generated SQL fails with: 

```sql
SELECT , "user"."id" AS "user__id", "user"."user_name" AS "user__user_name", "user"."last_name" AS "user__last_name", "user"."first_name" AS "user__first_name", "user"."email" AS "user__email", "user"."is_admin" AS "user__is_admin", "user"."role" AS "user__role", "user"."geom"AS "user__geom", "user"."password" AS "user__password", "user"."created_at" AS "user__created_at", "user"."updated_at" AS "user__updated_at" FROM "contract_users" AS "contract_user" LEFT JOIN "users" AS "user" ON "user"."id" = "contract_user"."user_id" WHERE ("contract_user".contract_id = 1 AND "contract_user".role = 'WORKER') ORDER BY random() LIMIT 4
```
```
2017/01/24 15:46:27 seeds.go:289: ERROR #42601 syntax error at or near "," (addr="127.0.0.1:5432")
```

Notice the isolated `,` after `SELECT`.

**Comment by vmihailenco** [comment]
Thanks for reporting - should be fixed in v5.2.8

**Comment by jeromenerf** [comment]
Fixed, thanks!
